tools/localizations/cn.csv

```diff
@@ -79,4 +79,6 @@ Aqara Wireless Mini Switch T1,Aqara 无线开关 T1
 "WallHero Switch (8 Way)",八位智能开关/场景面板
 "WallHero Outlet",智能墙面五孔插座
 "WallHero Remote Control (4-Inch)",语音场景(4寸）面板
+"Zemismart ZM02 Smart Curtain",Zemismart ZM02 智能窗帘
 "Zemismart ZM25C Smart Curtain",Zemismart ZM25C 智能窗帘
+
```

Comment:
this is the change that necessitated https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1808, so you'll need to include that wherever this is going for the deploy to succeed in CN.

Reply:
Thanks, I just added this to the cherry-pick 👍